configurer/path_linux.go

```
@@ -3,10 +3,19 @@
 package configurer
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
+
+	"github.com/pastelnetwork/pastel-utility/constants"
 )
 
+// GetHomeDir returns the home path for Linux OS.
+func GetHomeDir() string {
+	homeDir, _ := os.UserHomeDir()
```

Please don't ignore err